server/listeners/eventHandlers/domainEvents/releaseEventHandler.test.ts

```diff
@@ -151,10 +147,7 @@ describe('Release event handler', () => {
         restrictedPatient: false,
       },
     ])
-    prisonerService.getActiveHdcStatus.mockResolvedValue({
-      approvalStatus: 'APPROVED',
-      bookingId: '111',
-    })
+    prisonerService.isHdcApproved.mockResolvedValue(true)
```

Comment:
this was a bit horrible - it was coupled to the internals of the prisoner service as this method wasn't mocked. 

I've moved the prisonerservice over to be a simple mock and this now solely tests the interactions.
I've added proper unit tests to filling in missing coverage for this `isHdcApproved` method